_posts/blog/2018-01-24-modernizing-mlab.md

```diff
@@ -58,7 +61,7 @@ The design process and initial development of [ePoxy](https://github.com/m-lab/e
 
 ### Move to a Modern Kernel and TCP_INFO
 
-To remove M-Lab’s reliance on Web100, M-Lab plans to transition to the TCP_INFO kernel instrumentation. M-Lab has confirmed that the current TCP_INFO implementation in modern kernels provides equivalent measurements of the TCP stack. Work is beginning now to refactor or turn down all M-Lab tools that use web100 to use TCP_INFO, including: NDT, Sidestream, NPAD, and the Paris-Traceroute wrapper. Once this work is completed, we’ll test these new tools on our platform’s testing infrastructure while keeping web100 on all production servers.
+To remove M-Lab’s reliance on Web100, M-Lab plans to transition to the TCP_INFO kernel instrumentation. Basing our measurements on TCP_INFO kernel instrumentation will allow M-Lab to support the latest version of TCP (called [BBR TCP](https://queue.acm.org/detail.cfm?id=3022184)){:target="_blank"} that will allow download tests to converge much more quickly at high speeds. M-Lab has confirmed that the current TCP_INFO implementation in modern kernels provides equivalent measurements of the TCP stack. Work is beginning now to refactor or turn down all M-Lab tools that use web100 to use TCP_INFO, including: NDT, Sidestream, NPAD, and the Paris-Traceroute wrapper. Once this work is completed, we’ll test these new tools on our platform’s testing infrastructure while keeping web100 on all production servers.
```

Comment:
Change to `M-Lab plans to transition its hosted experiments to use the netlink socket and the TCP_INFO ....`